Kalela/Level-Up
# #156185242 Unit and UI tests

# What does this PR do?
Sets up initial unit and UI tests for the android level up project.

### How should this be manually tested?
1. Clone the repo.
2. cd into the base directory of the project.
2. Run ``` ./gradlew test ```  and ``` ./gradlew createDebugCoverageReport ``` to generate the coverage report.

NB: Check coverage report in app/build/reports/coverage/index.html.
![image](https://user-images.githubusercontent.com/33759757/43087868-8fd2d442-8ea9-11e8-8bbf-1ccd20c9c89b.png)



### What are the relevant pivotal tracker stories?
156185242
Unit Test your Code
157354404
UI Test your Code
